smoke/src/main/java/com/adobe/cq/cloud/testing/it/smoke/CreatePageAsAuthorUserIT.java

```diff
@@ -109,4 +117,49 @@ public void testCreatePageAsAuthor() throws InterruptedException, ClientExceptio
         }
     }
 
+    private SlingHttpResponse createTestPage(String pageName, int authRetries) throws ClientException, InterruptedException {
+        Supplier<SlingClient> creatorSupplier = cqBaseClassRule.authorRule::getAdminClient;
+        CQSecurityClient securityClient = creatorSupplier.get().adaptTo(CQSecurityClient.class);
+        CQPermissions permissionsObj = new CQPermissions(securityClient);
+
+        // check if the parent path of the test page is not blocked for "everyone" group, update the test group permissions if needed
+        JsonNode everyonePerms = null;
+        try {
+            everyonePerms = permissionsObj.getPermissions("everyone", temporaryPage.getParentPath(),0, HttpStatus.SC_OK);
+        } catch (ClientException e) {
+            if (e.getHttpStatusCode() != HttpStatus.SC_NOT_FOUND){
+                throw e;
+            }
+        }
+        if (everyonePerms != null) {
+            JsonNode permsEntry = everyonePerms.at("/entries/0");
+            if (!permsEntry.get("read").asBoolean() || !permsEntry.get("modify").asBoolean() || !permsEntry.get("create").asBoolean()) {
+                LOG.info("Updating group permissions for {}", temporaryPage.getParentPath());
+                permissionsObj.changePermissions(groupRule.getGroupName(), temporaryPage.getParentPath(),
+                        true, true, true, false,false, false, false,
+                        HttpStatus.SC_OK);
+            }
+        }
+
+        // create test page
+        SlingHttpResponse response = userRule.getClient().createPageWithRetry(
+                pageName,
+                "Page created by CreatePageAsAuthorUserIT",
+                temporaryPage.getParentPath(),
+                "",
+                MINUTES.toMillis(1),
+                500,
+                HttpStatus.SC_OK,
+                HttpStatus.SC_UNAUTHORIZED
+        );
+
+        // retry in case user wasn't fully set up on time
+        if (response.getStatusLine().getStatusCode() == HttpStatus.SC_UNAUTHORIZED && authRetries > 0) {
+            LOG.info("Got response status {} while creating page {}, retrying...", HttpStatus.SC_UNAUTHORIZED, pageName);
+            SECONDS.sleep(1);
+            return createTestPage(pageName, --authRetries);
```

Comment:
why is this retry needed?
The createPageWithRetry will already repeat the things for 1 minute

if the group is not available within 1 minute, there are rather other issues
then it's fine to fail, same as today